controllers/trainingjob/trainingjob_controller_test.go

```diff
@@ -914,35 +601,117 @@ func createTrainingJob(k8sName, sageMakerName string, hasFinalizer bool) *traini
 			StoppingCondition: &commonv1.StoppingCondition{},
 		},
 		Status: trainingjobv1.TrainingJobStatus{
-			TrainingJobStatus: InitializingJobStatus,
+			TrainingJobStatus: controllers.InitializingJobStatus,
```

Comment:
Remove this status. k8s clients ignore it when creating, so it only serves to confuse readers.